lib/particular-calendars/canada.ts

```diff
@@ -1,88 +1,119 @@
 import { PatronTitles } from '../constants/martyrology-metadata';
 import { Precedences } from '../constants/precedences';
 import { CalendarDef } from '../models/calendar-def';
-import { Inputs } from '../types/calendar-def';
+import { Inputs, ParticularConfig } from '../types/calendar-def';
 import { Americas } from './americas';
 
 export class Canada extends CalendarDef {
   parentCalendar = Americas;
 
+  particularConfig: ParticularConfig = {
+    epiphanyOnSunday: true,
+    ascensionOnSunday: true,
+    corpusChristiOnSunday: true,
```

Comment:
For this, we could source [this](https://www.cccb.ca/wp-content/uploads/2023/05/Ordo_Pastoral_Notes.pdf#page=92).

Reply:
Good find for the source!!

Ouch, I find this property name `mostHolyBodyAndBloodOfChristOnSunday` quite long... But for consistency, we might not have a choice but to go ahead with it...

Reply:
Indeed, it is very long and indeed we don’t have much choice. :wink: